packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,235 @@
+-   setWaferMapQuadrant - changes the layout of the dies on the wafer map. - Calling this method should re-render the wafer map.
+-   disable() - renders the component dimmed out and turns off interactivity.
+-   focus() - adds a green box around the component when it has tab focus.
+-   render() - renders a `nimble-wafer-map` component.
+
+Events: The events mentioned below will all be handled internally by the nimble component and they will not be part of the public API. In the initial implementation the following events should be handled:
+
+-   Zoom in while hovering - this action gets executed whenever the mouse pointer hovers the `nimble-wafer-map` component and a wheel event (scroll up) gets triggered
+-   Zoom out while hovering - this action gets executed whenever the mouse pointer hovers the `nimble-wafer-map` component and a wheel event (scroll down) gets triggered
+-   Drag while zoomed - this event gets triggered whenever the `nimble-wafer-map` component is zoomed in (larger than 100%) and whilst the left mouse button is held the pointer moves to any direction within the wafer-map canvas
+-   Mouse hover - this event gets triggered whenever the mouse pointer is hovering any of the die elements within the wafer map. We only must detect this in the nimble component, proper handling will be done in the MicroStrategy wrapper. (Tooltip triggering)
+
+### Anatomy
+
+Shadow DOM:
+
+```html
+<template id="WaferMapTemplate">
+    <div class="WaferMapContainer">
+        <svg>
+            <g class="zoomContainer">
+                <g>
+                    <circle></circle>
+                    <circle></circle>
+                    <rect></rect>
+                </g>
+            </g>
+        </svg>
+        <div class="WaferMapArea">
+            <canvas />
+            <svg>
+                <rect></rect>
+            </svg>
+        </div>
+    </div>
+</template>
+```
+
+The first SVG element with the "zoomContainer" class attribute contains two circles and a rectangle.
+
+-   The first circle represents the outline of the wafer map component.
+-   The second circle, together with the rectangle is part of a solution which creates the illusion that there is a notch present on the wafer map frame.
+
+The second SVG element with the "WaferMapArea" class attribute represents the area where the dies get rendered.
+The rectangle present here is used as the "highlight" which allows us to identify individual dies in the canvas.
+
+-   Slot Names: (none)
+-   Host Classes: (none)
+-   Slotted Content/Slotted Classes: (none)
+-   CSS Parts: (none)
+
+### Angular integration
+
+The `nimble-wafer-map` component does not take user input directly from the webpage where it's embedded. All the inputs must be provided through the constructor of the component so in the current case there is no point using ControlValueAccessors.
```

Comment:
ControlValueAccessors are for Angular Form participation like for buttons, text fields, radio groups, etc. So we will not for a ControlValueAccessor but we will require an Angular directive to expose the typed properties (very straightforward boilerplate).

```suggestion
The `nimble-wafer-map` component will not participate in Angular Forms and does not need a ControlValueAccessor. It will have a standard Angular Directive to expose typed properties to Angular.
```